testing/src/test/java/org/motechproject/nms/testing/it/api/MobileAcademyControllerBundleIT.java

```diff
@@ -158,4 +192,469 @@ public void testSmsStatusInvalidFormat() throws IOException, InterruptedExceptio
         assertTrue(SimpleHttpClient.execHttpRequest(request, HttpStatus.SC_BAD_REQUEST, RequestBuilder.ADMIN_USERNAME, RequestBuilder.ADMIN_PASSWORD));
     }
 
+    /**
+     * setup MA course structure from nmsCourse.json file.
+     */
+    private JSONObject setupMaCourse() throws IOException {
+        MaCourse course = new MaCourse();
+        String jsonText = IOUtils
```

Comment:
It used to be 113 characters because github web diff had a line-wrap display bug that made it such that we had to horizontally scroll when lines were wider than 113 chars, but it seems fixed now.